.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: bash
+services: docker
+
+matrix:
+  include:
+    - os: linux
+      env: VERSION=0.2
+    - os: linux
+      env: VERSION=0.3 LATEST=true
+
+before_script:
+  - env | sort
+  - cd "$VERSION"
+
+script:
+  - |
+    (
+      set -Eeuo pipefail
```

Comment:
It would clean up this file and remove redundancy to just call the script. The script handles the latest tagging as well so that wouldn't be a problem. Only reason I kept it separate was so each image build gets its own Travis job. But you think it might be better/okay just to have one Travis job to build all images?

Reply:
Yes, I think that just one Travis job is ok for both images. Two jobs might be better, but I would prefer if the `travis.yml` file wouldn't contain bash code and instead just execute a script.
Any way, that's just my opinion. We can also keep it as is if you prefer two Travis jobs.

Reply:
I'll see if I can add support for an optional version arg to build-images.sh, which would remove th e bash scripting from .travis.yml while keeping as multiple jobs. If that doesn't work I'll just go to one Travis job as I agree that would be cleaner.